src/test/java/org/broadinstitute/hellbender/tools/walkers/markduplicates/AbstractMarkDuplicatesCommandLineProgramTest.java

```diff
@@ -251,7 +254,7 @@ public void testOpticalDuplicatesTheSameReadGroup() {
     public void testOpticalDuplicatesAndPCRDuplicatesOrientationDifference() {
         final AbstractMarkDuplicatesTester tester = getTester();
         tester.setExpectedOpticalDuplicate(0);
-        tester.addMatePair("RUNID:7:1203:2886:82292",  19, 19, 485253, 486253, false, false, true, true, "101M", "101M", true, false, false, false, false, DEFAULT_BASE_QUALITY, "1");  // duplicate
+        tester.addMatePair("RUNID:7:1203:2886:16838",  19, 19, 485253, 486253, false, false, true, true, "101M", "101M", true, false, false, false, false, DEFAULT_BASE_QUALITY, "1");  // duplicate
```

Comment:
Why did this one need to change?  If it resolves differently now, how were both we and picard passing before?  Or is this a gatk only test?

Reply:
So this test is different now because the tiebreaking has changed, specifically because of integer overflowing in the Y coordinate of this read. The point of this test is about orientation and optical duplicates (which it wasn't doing a good job of testing before anyway because it was failing for a different reason). The tiebreaking change caused a different read to be marked as duplicate. I could have reversed the value but it seems better to correct the test as now its actually testing what it purports to. Also, yes this does not exist in picard.